jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -1,41 +1,18 @@
 #!/bin/bash
 export STRICT="NO"
 source "${HOMEgfs}/ush/preamble.sh"
+WIPE_DATA="NO"
+DATA="${DATA:-${DATAROOT}/${RUN}ocnanal_${cyc}}"
```

Comment:
why can't this be:
DATA="${DATAROOT}/${RUN}ocnanal_${cyc}}"
There is no one upstream setting `DATA`

Reply:
It can be. I just copied what was there before.